charts/camunda-platform/charts/web-modeler/templates/deployment-webapp.yaml

```diff
@@ -52,9 +52,11 @@ spec:
           - name: OAUTH2_TOKEN_ISSUER
             value: {{ .Values.global.identity.auth.publicIssuerUrl | quote }}
           - name: KEYCLOAK_BASE_URL
-            value: {{ .Values.global.identity.auth.publicIssuerUrl | trimSuffix "/auth/realms/camunda-platform" | quote }}
+            value: {{ .Values.global.identity.auth.publicIssuerUrl | trimSuffix (print (.Values.global.identity.keycloak.contextPath | trimSuffix "/") .Values.global.identity.keycloak.realm) | quote }}
```

Comment:
I'd suggest creating a new named templated `camundaPlatform.keycloakBaseURL` and using it here and in [camundaPlatform.keycloakURL](https://github.com/camunda/camunda-platform-helm/blob/64067d64b5f4f726fb6b30396a8eeea2c9b1d1f7/charts/camunda-platform/templates/_helpers.tpl#L119).

Reply:
Hm, I don't think that this would work, because `camundaPlatform.keycloakURL` is used for backchannel communication with Keycloak in `camundaPlatform.issuerBackendUrl`.
However, the `KEYCLOAK_BASE_URL` here must always use the public URL that can be accessed in the browser.

Reply:
Ah I see, thanks for the clarification :+1: